src/components/progress-report/variance-explanation/variance-explanation.dto.ts

```diff
@@ -26,6 +26,9 @@ export abstract class ProgressReportVarianceExplanation {
 
   readonly report: ProgressReport & SetUnsecuredType<ID>;
 
+  @Field()
+  readonly deprecated: SecuredStringList;
```

Comment:
This isn't needed. The other one is sufficient

Reply:
[Here](https://github.com/SeedCompany/cord-api-v3/pull/2914/files#diff-3c01125b0bcbed569795854d33632f57f8f35e2dac24bd6090c2ee8015376b94R19). This DTO shape is for the actual saved value for each report, we don't need to track the options here or if they are deprecated.

Reply:
Oh I see